Calculator/Calculator/ViewController.swift

```diff
@@ -0,0 +1,333 @@
+//
+//  ViewController.swift
+//  Calculator
+//
+//  Created by Leidy Luna on 15/07/23.
+//
+
+import UIKit
+
+class ViewController: UIViewController {
+
+    
+    @IBOutlet weak var mainLabel: UILabel!
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        // Do any additional setup after loading the view.
+    }
+    
+    @IBAction func equal(_ sender: Any) {
+       
+       var aux = ""
+       var operationInArray: [String] = []
+        
+        for char in mainLabel.text! {
```

Comment:
can cause a crash is text is nil

Reply:
This way is better? 

var aux: String = ""
        var operationInArray: [String] = []
        
        for char in mainLabel.text ?? "0" {

Reply:
why setting a default value if you don't need it. you can use `if let` for that purpose